apiserver/facades/agent/provisioner/provisioner_test.go

```diff
@@ -1421,28 +1421,90 @@ func (s *withoutControllerSuite) TestContainerManagerConfigDefaults(c *gc.C) {
 	})
 }
 
-type withImageMetadataSuite struct {
-	provisionerSuite
-}
+func (s *withoutControllerSuite) TestWatchMachineErrorRetry(c *gc.C) {
```

Comment:
Some non-functional churn here due to method relocation, but having tests from different suite spread around was annoying.